src/main/java/org/sputnikdev/bluetooth/gattparser/spec/BluetoothGattSpecificationReader.java

```diff
@@ -221,6 +226,54 @@ public void loadExtensionsFromFolder(String path) {
         readCharacteristics(getFilesFromFolder(characteristicsFolderName));
     }
 
+    private static URL specResourceURL(URL original, String characteristicType) throws MalformedURLException {
```

Comment:
It is looking good, but I'd prefer to keep Java code convention to use 'verbs' for methods that do something, e.g. `getSpecResourceURL`